Description: skip unit tests that require a too recent version of libwps (xenial has 0.4.3)
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/writerperfect/qa/unit/WpftCalcFilterTest.cxx
+++ b/writerperfect/qa/unit/WpftCalcFilterTest.cxx
@@ -35,14 +35,9 @@ void WpftCalcFilterTest::test()
     {
         {"Numbers_2.numbers", REQUIRE_ETONYEK_VERSION(0, 1, 2)},
     };
-    const writerperfect::test::WpftOptionalMap_t aWpsOptional
-    {
-        {"Lotus123_3.123", REQUIRE_WPS_VERSION(0, 4, 5)},
-        {"Lotus123_98.123", REQUIRE_WPS_VERSION(0, 4, 6)},
-    };
 
     doTest("com.sun.star.comp.Calc.MWAWCalcImportFilter", "/writerperfect/qa/unit/data/calc/libmwaw/");
-    doTest("com.sun.star.comp.Calc.MSWorksCalcImportFilter", "/writerperfect/qa/unit/data/calc/libwps/", aWpsOptional);
+    doTest("com.sun.star.comp.Calc.MSWorksCalcImportFilter", "/writerperfect/qa/unit/data/calc/libwps/");
     doTest("org.libreoffice.comp.Calc.NumbersImportFilter", "/writerperfect/qa/unit/data/calc/libetonyek/", aEtonyekOptional);
     doTest("org.libreoffice.comp.Calc.StarOfficeCalcImportFilter", "/writerperfect/qa/unit/data/calc/libstaroffice/");
 }
--- a/unotest/source/cpp/filters-test.cxx
+++ b/unotest/source/cpp/filters-test.cxx
@@ -95,6 +95,10 @@ void FiltersTest::recursiveScan(filterSt
             if (sURL[nLastSlash+1] == '.')
                 continue;
 
+            // skip files that require a too recent version of libwps
+            if (sURL.match("Lotus123_3.123") || sURL.match("Lotus123_98.123"))
+                continue;
+
             if (
                 (sURL.match("BID", nLastSlash+1)) ||
                 (sURL.match("CVE", nLastSlash+1)) ||
